### ISSUE TITLE
studio(a11y): presentations should have better naming

### ISSUE THREAD

**Issue by donroyco** [deckgo/deckdeckgo#535]
## Current behaviour

When navigating through the presentations with MacOS built in [VoiceOver](https://help.apple.com/voiceover/mac/10.15/), it just speaks out the URLs leading to Google Firebase storage.

Reproducible steps:
1. Go to https://deckdeckgo.com/
2. Switch on VoiceOver (Cmd + F5)
3. Navigate with tab to the list of presentations

<img width="915" alt="MacOS VoiceOver speaks out URL" src="https://user-images.githubusercontent.com/1763537/70848610-98fd5400-1e74-11ea-87a0-f74be9e66dd0.png">

## Expected behaviour

It should speak out the title of the presentation (in combination with the author).

**Comment by peterpeterparker** [comment]
freak me, the `alt` tag is missing, sorry for that

thanks a lot @donroyco for these accessibility tests 🙏

**Comment by donroyco** [comment]
Would adding 

```html
aria-label={deck.data.meta.title}
``` 
 to the link element make sense? I am not sure if the meta title is always available.

**Comment by peterpeterparker** [comment]
It makes sense. Title is mandatory in the meta data.
